distribution/docs/src/main/resources/content/_architectures/_securityFramework/federated-identity.adoc

```diff
@@ -19,59 +19,31 @@ are guaranteed to have a Subject. However, when federating, {branding} proxies r
 Sources and this poses a problem. The requestor doesn't authenticate with federated Sources, but
 Sources still need to make authorization decisions.
 
-To solve this problem, {branding} uses federated identity. When performing any federated request
-(query, resource retrival, etc), {branding} attaches the requestor's SAML assertion to the outgoing
-request. The federated Source extracts the assertion and validates its signature to make sure it
-was generated by a trusted entity. If so, the federated Source will construct a Subject for the
-requestor and perform the request using that Subject. The Source can then make authorization
+To solve this problem, {branding} will use its own identity (PKI or configured username/password per
+source) or in the case of OAuth 2, ask the user
```

Comment:
❗ The `None/Dynamic` authentication type says it supports both PKI and SAML ECP:
```
None/Dynamic will still attempt PKI and/or SAML ECP authentication if available.
```
This section mentions PKI, basic, and OAuth, but is missing ECP
Also, if it's not addressed elsewhere, can we add docs detailing what happens with the Dynamic auth type, particularly how it decides which to use?

Reply:
I'm not seeing the none/dynamic thing anywhere. The short answer is that it no longer exists.

Reply:
It's in a bunch of the source configurations. For example: https://github.com/codice/ddf/blob/master/catalog/confluence/catalog-confluence-source/src/main/resources/OSGI-INF/metatype/metatype.xml#L25-L27

Reply:
I did not put ECP in there because that is still going to just use username/password or PKI, just through a different means.

Reply:
Ah, I see. I was only messing with the security docs themselves.

Reply:
Oh I see, that is on the source side. That isn't actually an option, it is just the absence of a choice represented in the metatype. The only real choice is basic auth or OAuth (for some sources). If you leave it blank, it'll just try to connect. We always send ECP headers, so if someone replies back with ECP then we would try it.

Reply:
That's exactly the information I was looking for. Imo it would be good to document since it's not clear what happens under the hood with that option selected (or if nothing else is selected)

Reply:
As for the PKI side of dynamic auth, does DDF always present a client cert?

Reply:
As long as the endpoint asks for it, it will.